CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+### citus v11.0.5 (August 1, 2022) ###
+
+* Avoids possible information leakage about existing users
+
+* Allows using `WITH HOLD` cursors with parameters
+
+* Fixes a bug that could cause failures in `INSERT INTO .. SELECT`
+
+* Fixes a bug that prevents pushing down `IN` expressions when using columnar
+  custom scan
+
+* Fixes a concurrency bug between creating a co-located distributed table and
+  shard moves
+
+* Fixes a crash that can happen due to catalog read in shmem_exit
```

Comment:
```suggestion
* Fixes a crash that can happen due to catalog read in `shmem_exit`
```

I am not sure if a user would know about `shmem_exit`. Should we paraphrase this one maybe?

Reply:
It's a bit hard to make this more user-friendly than it is, so preferring to leave this as is